### PR TITLE
Make ioctl be able to send raw transaction bytes 1319

### PR DESCRIPTION
**Pull request by lzxm160**
work on #1319

use:
ioctl action sendraw 0a590801101518e0a712220d3130303030303030303030303052400a13313534353530303030303030303030303030301229696f3165796e39746336743738327a78347a677933686774333268707a36743876377067663532347a124104ea8046cf8dc5bc9cda5f2e83e5d2d61932ad7e0e402b4f4cb65b58e9618891f54cba5cfcda873351ad9da1f5a819f54bba9e8343f2edd1ad34dcf7f35de552f31a41eb9037e0d5c71c48acf114b2e3d68fb5313873e73e02ce30eff8ace0e6f879ca36c6ed806bacc34aeef40aab4a8629e4acc7784f94fa4eda3bbed375abf183d400

### CODE REVIEW

File: cli/ioctl/cmd/action/action.go [hunk 1]
```diff
@@ -136,7 +138,27 @@ func execute(contract string, amount *big.Int, bytecode []byte) (err error) {
...
+	request := &iotexapi.SendActionRequest{Action: selp}
+	if _, err = cli.SendAction(ctx, request); err != nil {
+		if sta, ok := status.FromError(err); ok {
+			return fmt.Errorf(sta.Message())
+		}
+		return err
+	}
+	shash := hash.Hash256b(byteutil.Must(proto.Marshal(selp)))
+	fmt.Println("Action has been sent to blockchain.")
+	fmt.Printf("Wait for several seconds and query this action by hash: %s\n", hex.EncodeToString(shash[:]))
```
Comment (zjshen14) [inline cli/ioctl/cmd/action/action.go]: Try %x, which is hex format for bytes

File: cli/ioctl/cmd/action/action.go [hunk 2]
```diff
@@ -136,7 +138,27 @@ func execute(contract string, amount *big.Int, bytecode []byte) (err error) {
 		signer,
 	)
 }
+func sendRaw(selp *iotextypes.Action) error {
```
Comment (zjshen14) [inline cli/ioctl/cmd/action/action.go]: sendRaw -> sendAction?

File: cli/ioctl/cmd/action/actionsendraw.go [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+// actionSendRawCmd represents the action send raw transaction command
+var actionSendRawCmd = &cobra.Command{
+	Use:   "sendraw DATA",
+	Short: "Send raw action on IoTeX blokchain",
+	Args:  cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.SilenceUsage = true
+		act := &iotextypes.Action{}
+		actByte, err := hex.DecodeString(args[0])
```
Comment (zjshen14) [inline cli/ioctl/cmd/action/actionsendraw.go]: actByte -> actBytes

File: cli/ioctl/cmd/action/actionsendraw.go [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+)
+
+// actionSendRawCmd represents the action send raw transaction command
+var actionSendRawCmd = &cobra.Command{
+	Use:   "sendraw DATA",
+	Short: "Send raw action on IoTeX blokchain",
+	Args:  cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.SilenceUsage = true
+		act := &iotextypes.Action{}
```
Comment (zjshen14) [inline cli/ioctl/cmd/action/actionsendraw.go]: Move L24 to L28

File: cli/ioctl/cmd/action/action.go (hunk 1 above)
Comment (lzxm160) [reply cli/ioctl/cmd/action/action.go]: sure

Comment (lzxm160) [reply cli/ioctl/cmd/action/action.go]: hex.EncodeToString  make it string already

File: cli/ioctl/cmd/action/actionsendraw.go (hunk 4 above)
Comment (lzxm160) [reply cli/ioctl/cmd/action/actionsendraw.go]: sure

File: cli/ioctl/cmd/action/action.go (hunk 1 above)
Comment (zjshen14) [reply cli/ioctl/cmd/action/action.go]: Yes, I mean with %x you don't need to call hex.EncodeToString

**Pull request merged**